packages/stryker/src/mutators/BooleanSubstitutionMutator.ts

```diff
@@ -0,0 +1,30 @@
+import { Mutator, IdentifiedNode } from 'stryker-api/mutant';
+import { Syntax } from 'esprima';
+
+export default class BooleanSubstitutionMutator implements Mutator {
+  name = 'BooleanSubstitution';
+  private booleanValues: { [targetedOperator: string]: string } = {
+    'true': 'false',
+    'false': 'true'
+  };
+
+  applyMutations(node: IdentifiedNode, copy: <T>(obj: T, deep?: boolean) => T): IdentifiedNode[] {
+    let nodes: IdentifiedNode[] = [];
+    
+    // !a -> a
+    if (node.type === Syntax.UnaryExpression && node.operator === '!') {
+      let mutatedNode = copy(node.argument) as IdentifiedNode;
+      mutatedNode.nodeID = node.nodeID;
+      nodes.push(mutatedNode);
+    }
+
+    // true -> false or false -> true
+    if (node.type === Syntax.Literal && this.booleanValues[node.raw]) {
```

Comment:
I think `this.booleanValues` is not needed anymore. You can use `typeof node.value === 'boolean'` instead.